GeneratorInterface/Core/python/ExternalGeneratorFilter.py

```diff
@@ -1,9 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 class ExternalGeneratorFilter(cms.EDFilter):
-    def __init__(self, prod, _external_process_waitTime_ = cms.untracked.uint32(60), _external_process_verbose_ = cms.untracked.bool(False)):
+    def __init__(self, prod, _external_process_waitTime_ = cms.untracked.uint32(60), _external_process_verbose_ = cms.untracked.bool(False),
+                 _external_process_esModules_ = cms.vstring()):
```

Comment:
NOTE: although I called this `_external_process_esModules_`, technically the module names of any type of plugin could be passed and it would work. It only really makes sense for ES modules (and maybe Services for testing). If you'd rather, this could be changed to `_external_process_components_` or something else to capture what is technically possible rather than what I think is most reasonable.

Reply:
I like the proposal components as it might avoid confusion.